sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("PaloAltoNetworks")]
```

Comment:
Should be `PaloAltoNetworks.Cloudngfw`

Reply:
Namespace review is complete now - https://github.com/Azure/azure-sdk-pr/issues/892
The approved namespace is for .NET is "Azure.ResourceManager.PaloAltoNetworks.Ngfw".
I have regererated the code to reflect this change.